R/stat-.R

```diff
@@ -166,7 +166,7 @@ Stat <- ggproto("Stat",
     dropped <- non_constant_columns[!non_constant_columns %in% self$dropped_aes]
     if (length(dropped) > 0) {
       cli::cli_warn(c(
-        "The following aesthetics were dropped during statistical transformation: {.field {glue_collapse(dropped, sep = ', ')}}",
+        "The following aesthetics were dropped during statistical transformation: {.field {dropped}}.",
```

Comment:
```suggestion
        "The following aesthetics were dropped during statistical transformation: {.and {.field {dropped}}}.",
```

Reply:
it doesn't change anything here :)

``` r
x <- c("xx", "Yy")
cli::cli_text(
  "je {.field {x}}"
)
#> je xx and Yy
```

<sup>Created on 2023-12-04 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>